utils/yoke/src/macro_impls.rs

```diff
@@ -64,6 +64,30 @@ impl_copy_type!(bool);
 ///
 /// The [`ZeroCopyFrom`] impl will call `.clone()` since there is no borrowed version of this type. This
 /// will be cheap for `Copy` types.
+///
+/// For example, the following custom derive would fail to compile:
+///
+/// ```rust,compile_fail
+/// use yoke::{Yokeable, ZeroCopyFrom};
+/// use std::borrow::Cow;
+///
+/// #[derive(Yokeable, ZeroCopyFrom)]
+/// struct Foo<'a> {
+///    arr: [String; 12],
```

Comment:
Shouldn't this case be handled by `#[cloning_zcf]` or no?

Reply:
Yes, but that unconditionally clones, whereas OwnedYokeable lets you do this selectively per-field.

A thing that might be better is allowing `#[yoke(cloning_zcf)]` on individual _fields_. What if I remove OwnedYokeable and file an issue for that instead?

Reply:
Yes, I think that makes more sense. If we add new types to the crate, they should be things we want people to use, but this OwnedYokeable with a fake ZCF impl is generally something users should avoid. Annotations make it more clear that you're doing something against the grain, and it's more semantically correct, too.

Reply:
Removed, and opened https://github.com/unicode-org/icu4x/pull/1307 with the cloning_zcf expansion